eldritch/expansions/seaside/characters.py

```diff
@@ -0,0 +1,86 @@
+from eldritch import characters
+
+
+class Secretary(characters.Character):
```

Comment:
If you're adding the secretary here, can you also add her to names.js, and add the specials to the specials deck?